.github/scripts/hermetic_library_generation.sh

```diff
@@ -95,6 +95,14 @@ changed_libraries=$(python hermetic_build/common/cli/get_changed_libraries.py cr
   --current-generation-config-path="${generation_config}")
 echo "Changed libraries are: ${changed_libraries:-"No changed library"}."
 
+# do not generate showcase automatically
```

Comment:
We probably do want to generate showcase automatically? For example, if we update protoc version in the docker image, we do want the showcase module to be updated.

Reply:
I'll be testing automatic generation of showcase

Reply:
Confirmed automatic generation occurred in https://github.com/googleapis/sdk-platform-java/actions/runs/12901902434/job/35974671498